### ISSUE TITLE
bug: Not Working as intended (see Image)

### ISSUE THREAD

**Issue by Nicknamely** [brianhuster/live-preview.nvim#245]
### Did you read the documentation and check existing issues?

- [X] I have read all the documentations ([README](https://github.com/brianhuster/live-preview.nvim), [`:h livepreview`](https://github.com/brianhuster/live-preview.nvim/blob/main/doc/livepreview.txt), [Wiki](https://github.com/brianhuster/live-preview.nvim/wiki))
- [X] I have updated the plugin to the latest version before submitting this issue
- [X] I have searched the [existing issues](https://github.com/brianhuster/live-preview.nvim/issues) and [closed issues](https://github.com/brianhuster/live-preview.nvim/issues?q=is%3Aissue+is%3Aclosed) issues

### Neovim version (nvim -v)

0.10.2

### Operating system/version

Arch

### Describe the bug

![image](https://github.com/user-attachments/assets/7b8e5793-4c26-4861-a241-66516130f15c)

Page isnt being previewd, instead i get this (see image)

Browser: Zen

My config:
```lua
return {

  {
    "brianhuster/live-preview.nvim",
    dependencies = {
      "brianhuster/autosave.nvim", -- Not required, but recomended for autosaving and sync scrolling

      -- You can choose one of the following pickers
      -- "nvim-telescope/telescope.nvim",
      "ibhagwan/fzf-lua",
      -- "echasnovski/mini.pick",
    },
    opts = {},
  },
}
```
It sometimes works, but as soon as I update the file (or accidentally save it), the issue occurs. Occasionally, reloading the page also triggers the problem.

### Steps To Reproduce

1. Open an HTML file 
2. Start LivePreview

### Expected Behavior

Preview of the page.

### Health

```log
==============================================================================
livepreview: require("livepreview.health").check()

Check dependencies ~
- OK Nvim 0.10.2 is compatible with Live Preview
- OK `sh` is available
- WARNING `telescope` (optional) is not installed
- OK `fzf-lua` is installed
- WARNING `mini.pick` (optional) is not installed

Checkhealth server and process ~
- OK This Nvim process's PID is 33485
- OK Server is healthy on port 5500
- OK Server root: /home/nicknamely

Check your live-preview.nvim config ~
- {
    autokill = false,
    browser = "default",
    cmd = "LivePreview",
    dynamic_root = false,
    port = 5500,
    sync_scroll = false
  }

```
```


### Repro

```lua
  vim.env.LAZY_STDPATH = ".repro"
  load(vim.fn.system("curl -s https://raw.githubusercontent.com/folke/lazy.nvim/main/bootstrap.lua"))()
  
  require("lazy.minit").repro({
      spec = {
      {
          "brianhuster/live-preview.nvim",
          config = function()
              require("live-preview").setup({
                  -- Add your configuration here
  
              })
          end,
      },
  })

```

**Comment by brianhuster** [comment]
Hm, I still don't understand what you mean. What is your prob exactly?

**Comment by Nicknamely** [comment]
> The "Repro" section is there so that users have to make sure the problem they have with this plugin is not caused by another plugin or any other parts of their config, or if it is, users should know which plugin or config caused the problem. That's why it is compulsory. Also I don't use Lazyvim and have never used, so the info "On lazyVim latest" is meaningless to me. And it should be redundant with "Repro" section.

Sorry, I wasn’t sure what was expected of me. I made some edits and tried to provide as much information as I could. I also set up the repro environment and confirmed that the issue persists. That was something new to me, thanks.
This issue persists regardless of the browser (firefox or zen) I use. Is there anything else I can provide to help troubleshoot this?

**Comment by brianhuster** [comment]
Please describe the bug more thoroughly. I still don't understand what kind of bug you are encountering.

**Comment by Nicknamely** [comment]
The page isn't being previewed; instead, I get this (see image)."
![15-12-2024-1625-zen-beta](https://github.com/user-attachments/assets/62f7694f-9452-4d3a-872a-02220e55e304)

**Comment by brianhuster** [comment]
What kind of browser is it?

**Comment by brianhuster** [comment]
Can you share me the screenshot of Firefox?

**Comment by Nicknamely** [comment]
[zen browser](https://zen-browser.app/) It is firefox base.

> What kind of browser is it?

**Comment by Nicknamely** [comment]
![image](https://github.com/user-attachments/assets/dca867db-4784-445d-a66b-3985040add79)
On firefox.

**Comment by brianhuster** [comment]
This is very strange. I can't reproduce your bug. Can you send me your index.html?

**Comment by Nicknamely** [comment]
I've done some digging, and this happens when I restart LivePreview. Sorry about that. However, it still takes too long to load. I recorded 2 minutes, and it still didn’t even load.

**Comment by Nicknamely** [comment]
> This is very strange. I can't reproduce your bug. Can you send me your index.html?

Try closing the closing `LivePreview` while it still loading.
I have just waited 5 min and still didnt get a preview. Here the file

```html
<!doctype html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Card-Containner-Mosaic</title>
  </head>
  <body>
	  <h1> This is a title </h1>
	  <h2> This is another </h2>
	  aldkjsld
    <div class="container">
      <div class="container">
        <div class="card item1"></div>
      </div>
      <div class="container">
        <div class="card item2"></div>
      </div>
      <div class="container">
        <div class="card item3"></div>
      </div>
      <div class="container">
        <div class="card item4"></div>
      </div>
      <div class="container">
        <div class="card item5"></div>
      </div>
      <div class="container">
        <div class="card item6"></div>
      </div>
    </div>
  </body>
</html>
```

**Comment by Nicknamely** [comment]
Yeah, it seems the plugin isn’t functioning as expected. I have to manually reload the page for it to update, and it still takes a long time to load. There's no advantage to using the plugin when manually opening the localhost and viewing the page that way is faster.

**Comment by Nicknamely** [comment]
@brianhuster Did you manage to reproduce it ?

**Comment by brianhuster** [comment]
Yes, I try your code and it doesn't work at all, in both my plugin and the VSCode's livepreview extension. I tried removing all the div with class "container" and it works. 

But still, it seems like FIrefox browsers has bigger latency. I know how to fix, but maybe I'll do that tonight (GMT+7)

**Comment by brianhuster** [comment]
Please try the latest version
